src/tokenFactory/tokensFactory.ts

```diff
@@ -46,7 +49,7 @@ function signJWT(jwt: JwtToken, rawPrivateKey: string) {
 }
 
 function verifyJWT(jwt: string | SignedToken, rawPublicKey: string) {
-  return new TokenVerifier('ES256K', rawPublicKey).verify(jwt)
+  return new TokenVerifier('ES256K', PublicKeyUtils.getUncompressedFormatPublicKey(rawPublicKey)).verify(jwt)
```

Comment:
How did you test this function? Does the example https://github.com/alastria/alastria-identity-example/blob/develop/exampleTokens/1.tokenFactory.js work correctly?

Reply:
All the examples were working yep